com.mandelbulber.Mandelbulber.yaml

```diff
@@ -0,0 +1,115 @@
+app-id: com.mandelbulber.Mandelbulber
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+command: mandelbulber2
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
+cleanup:
+  #- /include
+  #- /lib/cmake
+  #- /share/doc
+  #- /share/applications
+  #- *.la
+  #- *.a
+
+modules:
+  - name: libpng12
+    rm-configure: true
+    config-opts:
+      - --disable-static
+      - --enable-maintainer-mode
+      - --with-libpng-compat
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/libpng/files/libpng12/1.2.59/libpng-1.2.59.tar.xz
+        sha256: b4635f15b8adccc8ad0934eea485ef59cc4cae24d0f0300a9a941e51974ffcc7
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} .;
+          
+  - name: gsl
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/gsl/gsl-2.6.tar.gz
+        sha256: b782339fc7a38fe17689cb39966c4d821236c28018b6593ddb6fd59ee40786a8
+
+  - name: opencl-headers
+    buildsystem: simple
+    build-commands:
+      - cp -av CL /app/include
+    sources:
+      - type: archive
+        url: https://github.com/KhronosGroup/OpenCL-Headers/archive/refs/tags/v2020.06.16.tar.gz
+        sha256: 2f5a60e5ac4b127650618c58a7e3b35a84dbf23c1a0ac72eb5e7baf221600e06
+  
+  - name: ocl-icd
```

Comment:
This is probably not needed (https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/master/elements/components/opencl.bst)